src/consts.py

```diff
@@ -0,0 +1,8 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""This module defines constants used throughout the Flask application."""
+
+FLASK_CONTAINER_NAME = "flask-app"
+FLASK_APP_PORT = 8000
```

Comment:
IMHO `FLASK_APP_PORT` should be defined in CharmState instead for better cohesion, and perhaps it should always be retrieved using the `flask_port` property

Reply:
Updated, thanks!

Reply:
`FLASK_SERVICE_NAME` and `FLASK_CONTAINER_NAME` can't be move to `charm.py` since it will cause a circular import